_client/src/UiManager.cpp

```diff
@@ -36,15 +38,15 @@ void UiManager::init()
     SPDLOG_TRACE("UiManager::init");
 
     initMainWindow();
```

Comment:
We don't need methods initMainWindow and initDialogManager
Just do initialization here simple as `m_mainWindow = new ...`

Reply:
done